ops/mainnet/prod/backend/config.tf

```diff
@@ -50,6 +50,9 @@ locals {
       "6778479" = {
         providers = ["https://rpc.ankr.com/gnosis"]
       }
+      "1818848877" = {
+        providers = ["https://rpc.linea.build"]
+      }
```

Comment:
wont we need a more stable rpc for the expected load?

Reply:
guess its more important for core

Reply:
Yes, as SDK server is only using public RPCs.